ding/utils/k8s_launcher.py

```diff
@@ -0,0 +1,96 @@
+import yaml
```

Comment:
Whether this `k8s_launcher.py` can be moved into `k8s_helper.py`

Reply:
ok, the two files can be merged